### ISSUE TITLE
Receiver discovery not working.

### ISSUE THREAD

**Issue by MattMckenzy** [Rafostar/playercast#12]
This software is exactly what I'm looking for, thanks for your hard work!

I'm having a bit of a hard time getting the Receiver to work however. Once running, I see "Playercast waiting for connnection...." and my sender app is not discovering playercast. I've tried completely turning off my Windows Firewall in case it was a port issue, but it didn't help. Does playercast support the Cast discovery protocol? I would love help figuring this out.

Thank you.

**Comment by Rafostar** [comment]
> Does playercast support the Cast discovery protocol?

Nope. This app is something I did long ago when I was mainly in need to "cast" media on my desktop into Raspberry Pi connected to my TV. The only thing its supports is streaming over network from one playercast instance running on some device as sender into another playercast instance running somewhere else as receiver.

If this is what you have been doing and it still did not work, then most likely firewall configuration on your Router device is blocking either discovery, file transfer or even both.
